META-INF/plugin.xml

```diff
@@ -53,35 +53,32 @@
             <li>1.6.4 Fixed Tabdir-16</li>
             <li>1.6.5 Merged Tabdir-16 into main</li>
             <li>1.6.7 (mod: vl) Fixup for Idea 202+</li>
+            <li>1.7.0 (mod: vl) Fixup for Idea 233+ (2023.3.2+), add empty path prefixes</li>
         </ul>
         ]]>
     </change-notes>
 
-    <!--This allow plugin to be installed on another IntelliJ Platform products-->
-    <depends>com.intellij.modules.lang</depends>
+    <!--This allows plugin to be installed on another IntelliJ Platform products-->
+    <depends>com.intellij.modules.platform</depends>
 
-    <application-components>
-        <!-- Add your application components here -->
-    </application-components>
-
-    <project-components>
-        <component>
-            <implementation-class>ru.crazycoder.plugins.tabdir.ProjectConfigRegistrator</implementation-class>
-        </component>
-    </project-components>
 
     <actions>
         <!-- Add your actions here -->
     </actions>
 
     <extensions defaultExtensionNs="com.intellij">
         <applicationService serviceImplementation="ru.crazycoder.plugins.tabdir.configuration.GlobalConfig"/>
-        <applicationConfigurable instance="ru.crazycoder.plugins.tabdir.configuration.GlobalConfigConfigurable"/>
+        <applicationConfigurable instance="ru.crazycoder.plugins.tabdir.configuration.GlobalConfigConfigurable"
+                                 displayName="Tabdir"/>
 
-        <projectService serviceImplementation="ru.crazycoder.plugins.tabdir.configuration.ProjectConfig"/>
+        <projectService serviceImplementation="ru.crazycoder.plugins.tabdir.TitleFormatter"/>
+        <projectConfigurable
```

Comment:
Am I right that this block replaces ProjectConfigRegistrator?

Reply:
yes, this seems to be  a new IDEA way to register configuration services